docs/en/observability/observability-ai-assistant.asciidoc

```diff
@@ -50,26 +50,28 @@ To set up the AI Assistant:
 * https://platform.openai.com/docs/api-reference[OpenAI]
 * https://learn.microsoft.com/en-us/azure/cognitive-services/openai/reference[Azure OpenAI Service]
 
+<<<<<<< HEAD
 . From *{stack-manage-app}* -> *{connectors-ui}* in {kib}, create a {kibana-ref}/gen-ai-action-type.html[Generative AI connector]. 
+=======
+. From *{stack-manage-app}* -> *{connectors-ui}* in {kib}, create a {kibana-ref}/openai-action-type.html[OpenAI connector].
+>>>>>>> 79fca6af ([Docs] Remove connector info for ingesting external data (#3378))
```

Comment:
I vaguely remember reviewing a PR that updated the URL to the AI action. You should just be able to resolve the conflict like this: 

```suggestion
. From *{stack-manage-app}* -> *{connectors-ui}* in {kib}, create a {kibana-ref}/openai-action-type.html[OpenAI connector].
```